docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 hawkular:
-  image: "hawkular/hawkular-services:latest"
+  image: "hawkular/hawkular-services:devel"
```

Comment:
Right now, it's the only way to make tests pass since there isn't a release with latest hawkular-commons code.
But, I'm also not sure about using devel. I both like the idea and dislike it :speech_balloon: